src/components/Oops/Oops.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components/macro'
+
+export default function Oops() {
```

Comment:
A better name would probably be `ErrorMessage` or something like that.

Reply:
Renamed it. 😅